src/Controller/AccountController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Controller;
+
+use App\Repository\ArticleRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Http\Attribute\IsGranted;
+
+#[IsGranted("IS_AUTHENTICATED_FULLY")]
+
+class AccountController extends AbstractController
+{
+  /**
+   * Реализует отображение личного кабинета пользователя.
+   */
```

Comment:
```suggestion
  /**
   * @param ArticleRepository $articleRepository
   * @return Response
   */
```

Аннотации должны выглядеть вот таким вот образом.
Аннотации !== комментарии